eng/slngen.targets

```diff
@@ -10,4 +10,7 @@
     <!-- Include CompilerServices.Unsafe as it's a transitive test dependency. -->
     <IncludeInSolutionFile Condition="'$(IsNETCoreAppRef)' == 'true' and '$(MSBuildProjectName)' == 'System.Runtime.CompilerServices.Unsafe'">true</IncludeInSolutionFile>
   </PropertyGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <SlnGenCustomProjectTypeGuid Include=".ilproj" ProjectTypeGuid="{F571AB99-FB84-49F4-A0DF-F27AA55F3F3F}" />
```

Comment:
I assume you generated a new GUID for this. Is that correct?

Reply:
Do we want to run slngen and update the .sln files for this?

Reply:
yes, this is a new GUID generated with VS. Let's try to not update the solution files too often. A good time would be after the CoreLib shared project reference irritation is mitigated and optionally after https://github.com/microsoft/slngen/issues/193.